packages/engine/src/avatar/functions/avatarFootHeuristics.ts

```diff
@@ -53,39 +53,39 @@ const speedMultiplier = 2
 const footRotationOffset = new Quaternion()
   .setFromAxisAngle(V_100, Math.PI / 2)
   .multiply(new Quaternion().setFromAxisAngle(V_010, Math.PI))
-  .multiply(new Quaternion().setFromAxisAngle(V_100, -Math.PI / 6))
 
 //step threshold should be a function of leg length
 //walk threshold to determine when to move the feet back into standing position, should be
 export const setIkFootTarget = (localClientEntity: Entity, delta: number) => {
   const userID = getComponent(localClientEntity, UUIDComponent)
 
-  const ikTargetLeftFoot = UUIDComponent.getEntityByUUID((userID + ikTargets.leftFoot) as EntityUUID)
-  const ikTargetRightFoot = UUIDComponent.getEntityByUUID((userID + ikTargets.rightFoot) as EntityUUID)
-  if (!ikTargetLeftFoot || !ikTargetRightFoot) return
+  const leftFootEntity = UUIDComponent.getEntityByUUID((userID + ikTargets.leftFoot) as EntityUUID)
+  const rightFootEntity = UUIDComponent.getEntityByUUID((userID + ikTargets.rightFoot) as EntityUUID)
 
-  const leftFootTargetBlendWeight = AvatarIKTargetComponent.blendWeight[ikTargetLeftFoot]
-  const rightFootTargetBlendWeight = AvatarIKTargetComponent.blendWeight[ikTargetRightFoot]
+  if (!leftFootEntity || !rightFootEntity) return
+  AvatarIKTargetComponent.blendWeight[leftFootEntity] = 1
```

Comment:
I assume these were left in from testing. These will cause the ik solve to always run regardless of being in an immersive session.

Reply:
oh whoops, yes